<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1320</width>
    <height>836</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Quriel - Experimental Music Player</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>760</y>
      <width>561</width>
      <height>38</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="previous">
       <property name="text">
        <string>&lt;&lt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="play">
       <property name="text">
        <string>Play</string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QRadioButton" name="shuffle">
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Shuffle</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pause">
       <property name="text">
        <string>Pause</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="next">
       <property name="text">
        <string>&gt;&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>720</y>
      <width>441</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QSlider" name="durationSlider">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>50</y>
      <width>331</width>
      <height>621</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="clearPlayListButton">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="moveTrackUp">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>▲</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="moveTrackDown">
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>▼</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="savePlayListButton">
         <property name="text">
          <string>Save Playlist</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeFromQueue">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListView" name="playlistView"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>921</width>
      <height>621</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1,3">
       <item>
        <widget class="QPushButton" name="insertPlaylist">
         <property name="text">
          <string>Queue Playlist </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removePlaylist">
         <property name="text">
          <string>Delete Playlist</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="randomQuote">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(217, 249, 255);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QTableView" name="songs">
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QListView" name="playlistList"/>
     </item>
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QPushButton" name="addTrack">
         <property name="text">
          <string>Add Track(s)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="removeTrack">
         <property name="text">
          <string>Remove Track(s)</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>670</y>
      <width>441</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="trackInformationGrid">
     <item row="0" column="0" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="trackInformation">
       <property name="minimumSize">
        <size>
         <width>150</width>
         <height>22</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Calibri</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1320</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>Files</string>
    </property>
    <addaction name="addFiles"/>
   </widget>
   <addaction name="menuFiles"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="addFiles">
   <property name="text">
    <string>Add...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
